pkg/conduit/pipelines_parser.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package conduit wires up everything under the hood of a Conduit instance
+// including metrics, telemetry, logging, and server construction.
+// It should only ever interact with the Orchestrator, never individual
+// services. All of that responsibility should be left to the Orchestrator.
```

Comment:
We already have a package doc in `runtime.go` (probably where you copied it from), no need to add it here.

Reply:
thought that I only copied the license :\

Reply:
@maha-hajja I can suggest using GoLand's live templates. I've saved the Meroxa copyright notice as `lcns`, and whenever I need it, I type `lcns` and GoLand inserts it.

Reply:
@hariso @lovromazgon thank you for the ideas 👀